recipe/meta.yaml

```
@@ -22,15 +25,16 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - cmake >=3.8
-    - ninja
+    - cmake >=3.18
+    # we currently don't have ninja 1.11 but it's building and passing tests
```

not sure of the relevance of this comment, i.e. why's this information pertinent? also it's information that will presumably be outdated very quickly. Seems like there's some info for a future engineer you wanted to convey, though, just wondering what that would be?

---

I was asked to add it. What I can do is remove the comment and add the information into the PR.

---

Oh I think I get it, it was just a bit unclear to me.
```suggestion
    # we currently don't have ninja 1.11, which upstream specifies. However, this feedstock builds and passes tests with v1.10.2
```

---

Should I add this comment back in? In my recent commit I removed it, and added it to the PR.

---

I would keep the comment in the recipe, it might be missed in the future if it's in the PR description.

---

Added it back in!